Formula/f/flint.rb

```diff
@@ -21,25 +21,39 @@ class Flint < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:   "796f363a97eb42772140b94d31a9eaccfb2acf445175639aba7869250b293942"
   end
 
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "libtool" => :build
   depends_on "gmp"
   depends_on "mpfr"
-  depends_on "ntl"
-
   uses_from_macos "m4" => :build
 
   def install
+    # to build against NTL
```

Comment:
NTL doesn't appear anywhere else in the formula now, is this correct?

Reply:
Yes. We have a header-only interface.

Reply:
and the NTL interface header is a C++ header